lib/createJestRunner.ts

```diff
@@ -68,14 +68,16 @@ export default function createRunner<
           );
     }
 
-    _createInBandTestRun(
+    async _createInBandTestRun(
       tests: Array<Test>,
       watcher: TestWatcher,
       onStart: OnTestStart,
       onResult: OnTestSuccess,
       onFailure: OnTestFailure,
       options: TestRunnerOptions,
     ): Promise<void> {
+      const runner = (await import(runPath)).default;
```

Comment:
Hm.. I still see `require` in the lates version. Is this because of Babel: 

```js
const runner = (await Promise.resolve(`${runPath}`).then(s => _interopRequireWildcard(require(s)))).default;
```

Reply:
good catch 👍